data/json/npcs/exodii/exodii_merchant_definitions.json

```diff
@@ -64,5 +64,14 @@
     "name": { "str": "Rubik's PA md. 68 Battle Rifle" },
     "description": "The most popular gun to use the 12.3ln cartridge was, of course, the PA md. 71.  Its predecessor, the md. 68, was viewed by many as a sort of failure: although it was reliable and powerful, it was too heavy to be used as a good infantry weapon, and not really heavy enough to be a good support gun.  Enough were made, though, that during the zombie apocalypse, it gained a great deal of resurgent popularity as a light emplacement gun that used readily available ammunition.  It perfectly served the purposes of the Exodii, who had far less concern about its unwieldiness.\n\nThis particular rifle has been painted a dusty pink color and has a picture of a smiling eel twisting along the barrel.",
     "extend": { "flags": [ "TRADER_KEEP_EQUIPPED", "NO_TURRET" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "rubik_translate",
+    "name": { "str": "Rubik Personal Translator Access" },
+    "points": 0,
+    "description": "You can understand what Rubik means to say.",
+    "valid": false,
```

Comment:
Probably shouldn't actually show up on the mutations menu, even if it is a mutation under the hood. I would set `player_display` to false.

Reply:
Fair call